### ISSUE TITLE
Docs(?): Explicitly state that only the first font specified in a rule is used

### ISSUE THREAD

**Issue by JakeIsMeh** [nuxt/fonts#172]
I believe this, and a combination of having Inter locally installed caused https://github.com/nuxt/nuxt.com/pull/1589.

**Comment by danielroe** [comment]
Yes, this should be specified. Basically font-face rules are generated for the first font in a `font-family` and subsequent fonts are used to create fallbacks.
